client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -67,10 +67,12 @@ const MetadataEditor = createClass({
 
 	getRedditLink : function(){
 		const meta = this.props.metadata;
+
+		const shareLink = (meta.googleID ? meta.googleId + meta.shareId : meta.shareId);
```

Comment:
Just a slight change.

```suggestion
		const shareLink = meta.googleID ? meta.googleId + meta.shareId : meta.shareId;
```

Or.... if you think this is cleaner:

```suggestion
		const shareLink = (meta.googleID || '') + meta.shareId;
```

Reply:
I think both have their merits, but the second option is cleaner.